src/App.tsx

```diff
@@ -1,9 +1,19 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './App.scss';
+// import { event } from 'cypress/types/jquery';
 import { MoviesList } from './components/MoviesList';
 import moviesFromServer from './api/movies.json';
 
 export const App: React.FC = () => {
+  const [query, setQuery] = useState('');
+
+  const sortMovies:Movie[] = moviesFromServer.filter(
```

Comment:
Change the variable name 'sortMovies' to 'visibleMovies' to match the task description.
![image](https://github.com/mate-academy/react_movies-list-filter/assets/99403198/c1e2a959-7fb6-4281-881e-46891ccd65ba)